### PR TITLE
Dashboard readme

### LINKED ISSUE #137: Dashboard - README
Add basic details needed for the README that can then be formatted

### PR DESCRIPTION
**Pull request by alina-101**
Basic info needed for the README to then format

closes #137

### CODE REVIEW

File: dashboard/README.md [hunk 1]
```diff
@@ -1,7 +1,23 @@
 # Dashboard!!
```
Comment (e-lemma) [inline dashboard/README.md:1]: Double exclamation marks,  unprofessional.

File: dashboard/README.md [hunk 2]
```diff
@@ -1,7 +1,23 @@
...
 
 - Home page
+- Artists page
 - Locations page
-- Subscribe to emails
+- Subscriptions page
+- Tags page
```
Comment (e-lemma) [inline dashboard/README.md:7]: I'm assuming a section addressing each page will be part of the finished `README.md`?

File: dashboard/README.md (hunk 1 above)
Comment (alina-101) [reply dashboard/README.md:1]: Pedro can format this as needed.

File: dashboard/README.md (hunk 2 above)
Comment (alina-101) [reply dashboard/README.md:7]: Yes there will be.

**Review by e-lemma — APPROVED** [review]
OK to merge.

**Pull request merged**